pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1932,11 +1932,16 @@ func QuantityToByte(quantity resource.Quantity) (api.Memory, error) {
 }
 
 func QuantityToMebiByte(quantity resource.Quantity) (uint64, error) {
-	q := int64(float64(0.953674) * float64(quantity.ScaledValue(resource.Mega)))
-	if q < 0 {
-		return 0, fmt.Errorf("Quantity '%s' must be greate tan or equal to 0", quantity.String())
+	bytes, err := QuantityToByte(quantity)
+	if err != nil {
+		return 0, err
+	}
+	if bytes.Value == 0 {
+		return 0, nil
+	} else if bytes.Value < 1048576 {
+		return 1, nil
 	}
-	return uint64(q), nil
+	return uint64(float64(bytes.Value)/1048576 + 0.5), nil
```

Comment:
why does this need to round up now?

Reply:
I think it is surprising that if I specify 2.999 MiB that it gets rounded down to 2Mi. My thinking was that this is closer to the actual intention. While it does in practice probably not matter much.

Reply:
Btw. I think that I found out that the bandwidth parameter is not properly passed to libvirt and that it is never effective. It is just a one liner change to enable this, but I want to discuss it in a separate PR, since this practically means that right now there are no restrictions and if I just quietly fix it we may surprise existing users.

Reply:
@davidvossel removed all bandwith related changes. It now only fixes numa. Moving all other fixes to a dedicated PR.